frontend/src/app/dora/components/cycle-time/cycle-time.component.html

```diff
@@ -6,7 +6,7 @@
     <app-cycle-item
       *ngFor="let cycleItem of jiraColumnsForm.controls; let i = index"
       [metricsForm]="metricsForm"
-      [groupName]="i"
+      [groupName]="i + ''"
```

Comment:
the type conversion is too implicit, could we use some other explicit way instead, such as `i.toString()`

Reply:
fixed that to i.toString().